xml/selinux.xml

```diff
@@ -77,24 +80,21 @@
 
   <para>
    The interesting thing about this hack is that it occurred on a server
-   where nothing was really wrong. All permissions where set OK, but the
-   intruder had managed to get into the system. What becomes clearly evident
-   from this example is that in some cases additional security is
-   needed&mdash;a security that goes beyond what is offered by using
-   &selnx;. As a less complete and less complex alternative, &aa; can be
-   used.
+   where nothing was really wrong. All file permissions were set correctly,  
+   but the intruder still managed to get into the system. This example  
+   demonstrates that in some cases additional security is needed.
   </para>
 
   <para>
-   &aa; confines specific processes in their abilities to read/write and
-   execute files (and other things). Its view is mostly that things that
-   happen inside a process cannot escape.
-  </para>
+   &aa; is a less complete and less complex alternative. &aa; attempts
```

Comment:
```suggestion
   &aa; is a less-complete and less-complex alternative. &aa; attempts
```

Reply:
Resolved by removing that bit :)

Reply:
All I'd say is that in case people were coming to SLE from Ubuntu, AppArmor is the default on Ubuntu and is used in Snappy AFAIK. So 'buntu types might be more familiar with it. It may be weaker but it's easier; from what I have read, it's so easy to break stuff with SELinux that most people just turn it off completely. AppArmor thereby wins because people actually use it.